### PR TITLE
fixed for non-regular primary key

### PR DESCRIPTION
**Pull request by devvit**
solve #33

### CODE REVIEW

**Comment by izelnakri** [comment]
This makes sense, thank you! One question before I merge this, why do you think my uuid tests didnt catch this? I guess our test coverage doesn't cover this bug? 

Could you look into this as well? I think we should release this fix along with its relevant test case.

File: lib/paper_trail.ex [hunk 1]
```diff
@@ -322,7 +322,7 @@ defmodule PaperTrail do
     %Version{
       event: "update",
       item_type: changeset.data.__struct__ |> Module.split() |> List.last(),
-      item_id: changeset.data.id,
+      item_id: get_model_id_from_changeset(changeset),
```
Comment (izelnakri) [inline lib/paper_trail.ex]: This could simply be get_model_id(changeset.data) instead of a new function I think

**Comment by devvit** [comment]
Yeah, the function is unnecessary, removed.

I'm working for a legacy database that there are some non-regular named fields, hoping to be useful to others :)

**Comment by izelnakri** [comment]
This is very nice thank you!

**Pull request merged**